README.md

```diff
@@ -1,38 +1,53 @@
 # ctb-backend
 
-Jak to uruchomić?
+## How to run
 
-Najpierw w celu zbudowania obrazu wykonaj polecenie:
+At the beginig you must create an image:
 ```
 docker build . -t ctb
 ```
 
-Następnie w celu uruchomienia kontenera:
+The next step is starting container. For that you must type:
 ```
 docker run -d -p 8080:8080 ctb
 ```
 
-Wtedy bedzie można wejść (np przez przeglądarkę) na adres: `https://localhost:8080/`
+And then you should be able to open (in your browser) address: `https://localhost:8080/`.
 
-W celu wyłączenia serwera można wpisać:
+To shutdown the server, type:
```

Comment:
It would be great to split the whole section of shutting down to another `##` (h2) just like we use it for launching a container instance.